chef/automate
# Expose Hab Svc health check output in Applications APIs

### :nut_and_bolt: Description: What code changed, and why?

In #2368 we started to ingest the output of hab health checks into the database. In this change we expose this information via the applications APIs.

### :chains: Related Resources

#1232 

### :+1: Definition of Done

Needs integration tests!

### :athletic_shoe: How to Build and Test the Change

* Build automate-gateway and applications service
* Create sample data, e.g., with `applications_populate_database`
* When viewing data in the UI, you can see the health check outputs in the services responses, for example  https://a2-dev.test/api/v0/applications/service-groups/6ec2a554e23f582efa1e2433db5390d417116583cf6791b780ca563799f10fc3?pagination.page=1&pagination.size=25  includes this for an individual service:

```
            "health_check_result": {
                "stdout": "",
                "stderr": "curl: (7) Failed to connect to localhost port 443: Connection refused",
                "exit_status": 2
            }

```

### :white_check_mark: Checklist
